### PR TITLE
Translate transactional emails to Portuguese

### PR DESCRIPTION
**Pull request by johnfisherman**
Create new transactional emails in Portuguese.

### CODE REVIEW

File: src/core/data/email/email-exists-login_pt.yfm [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+</p>
+<p><br /></p>
+<p>Nesse mesmo link vai poder escolher uma nova password, caso se tenha esquecido dela.</p>
+<p><br /></p>
+<p>
+  <b>Nota:</b> Os detalhes de pagamento que inseriu não foram gravados.
+  Assim que fizer login vai poder mudar os seus detalhes na hora.
+</p>
+<p><br /></p>
+<p>Best wishes</p>
```
Comment (wpf500) [inline src/core/data/email/email-exists-login_pt.yfm:23]: This looks distinctly like English to me!

**Review by wpf500 — CHANGES_REQUESTED** [review]
Thank you!! There are a few best wishes left then it's ready to go :)

**Comment by johnfisherman** [comment]
Fixed!

**Comment by wpf500** [comment]
Nice one, merging :smile:

**Pull request merged**